src/w3d/renderer/ddsfile.cpp

```diff
@@ -592,6 +592,7 @@ bool DDSFileClass::Load()
 
     if (data == 0) {
         captainslog_warn("DDSFile '%s' appears to contain no data, size is %d.", m_name, fp->Size());
+        fp->Close();
```

Comment:
auto_file_ptr should indirectly be doing that?

Reply:
No it does not.

Reply:
right, comment as bugfix

Reply:
Original auto_file_ptr didn't call Close, we should keep it that way.

Reply:
I have reverted change to DDSFileClass::Load and will make a new Pull for `auto_file_ptr`

Reply:
I have taken another look at `auto_file_ptr` and `GameFileClass` in debugger. It does call `Close` on destruction via virtual destructor:

```c++
GameFileClass::~GameFileClass()
{
    Close();
}
```

So this needs no change.